src/main/java/com/db/db/volunteers/model/Form.java

```diff
@@ -9,30 +9,62 @@ public int getLocalGovId() {
         return localGovId;
     }
         
+    /**
+     * @return the pollingUnitVolunteerStats
+     */
+    public Iterable<PollingUnitVolunteerStats> getPollingUnitVolunteerStats() {
```

Comment:
Do we really need to utilise the **form for storing Polling Unit data** or will this be useful with a a **paged implementation**?